pkg/services/shorturls/short_url_service_test.go

```diff
@@ -19,22 +18,24 @@ func TestShortURLService(t *testing.T) {
 
 		service := ShortURLService{SQLStore: sqlStore}
 
-		uid, err := service.CreateShortURL(context.Background(), user, refPath)
+		newShortURL, err := service.CreateShortURL(context.Background(), user, refPath)
 		require.NoError(t, err)
-		assert.NotEmpty(t, uid)
+		require.NotNil(t, newShortURL)
+		require.NotEmpty(t, newShortURL.Uid)
 
-		path, err := service.GetFullURLByUID(context.Background(), user, uid)
+		existingShortURL, err := service.GetShortURLByUID(context.Background(), user, newShortURL.Uid)
 		require.NoError(t, err)
-		assert.NotEmpty(t, path)
-		assert.Equal(t, refPath, path)
+		require.NotNil(t, existingShortURL)
+		require.NotEmpty(t, existingShortURL.Path)
```

Comment:
I think this one is superfluous, since the next check will cover the same ground.